saleor/checkout/test_checkout.py

```diff
@@ -36,13 +36,13 @@ def test_checkout_clear_storage():
     checkout.storage['new'] = 1
     checkout.clear_storage()
     assert checkout.storage is None
-    assert checkout.modified == True
+    assert checkout.modified is True
```

Comment:
Admittedly I changed this because flake8 recommended is, and I always test True/False/None with "is".

If all you care is that it's "truthy", then sure, that's a better move.

True and False are constants in the builtins, so AFAIK they _are_ singletons like None.